ElrondNetwork/elrond-go
# Split miniblocks based on type txs scs

* Implemented splitting mechanism for mini blocks "from me" based on the type of transactions they contain inside (non smart contract calls txs or smart contract calls txs)
* Added ScheduledMiniBlocksEbableEpoch flag
* Sketched the implementation for adding scheduled mini blocks in the proposed block

**How to be tested**

Start a normal system test with txs and sc calls
In epoch 5 scheduled mini blocks mechanism will be activated.
After epoch 5 will passed we should check the log files (some messages like: "createScheduledMiniBlocks has been started", "createScheduledMiniBlocks has been finished" should be found)
